runtime/recipe/particle.js

```diff
@@ -48,10 +48,10 @@ export class Particle {
     let map = new Map();
```

Comment:
TODO(shans): evaluate whether this is the appropriate context root for cloneWithResolution

Reply:
Added :)